benchmarks/benchmark_moe.py

```
@@ -3574,6 +3603,96 @@ def ref_launch() -> None:
+            fused_sum_buffers = make_backend_w4a16_buffers(
+                backend_w4a16_weights,
+                m=batch_size,
+                topk=spec.top_k,
+                dtype=torch.bfloat16,
+                device=device,
+            )
+            fused_sum_output_storage = torch.empty_like(x)
+
+            def run_fused_sum_comparison() -> torch.Tensor:
+                return w4a16_moe(
+                    x,
```

Ruff (0.16.1)

[warning] Function definition does not bind loop variable `x`

(B023)